_includes/tutorials/ksql-nested-json/ksql/markup/dev/create-financial-transaction-stream.adoc

```diff
@@ -0,0 +1,30 @@
+The first thing we do is to create a stream `TRANSACTION_STREAM` based off stock purchase transactions topic `financial_txns`. Within the create `STREAM` statement, you'll use a `STRUCT` keyword to define each nested object.  The `STRUCT` keyword provides a mechanism to navigate to the desired fields in your query.
```

Comment:
Should this be 
> Within the `CREATE STREAM` statement... 
?

Reply:
Feel free to shoot this down but I think it would be helpful to give a definition to what a struct is.
>A struct is a ksqlDB data type that represents strongly typed structured, or nested, data. A struct is an ordered collection of named fields that have a specific type. [pulled directly from [ksqlDB docs](https://docs.ksqldb.io/en/latest/developer-guide/syntax-reference/#struct)]
As mentioned earlier, structs won't work in all scenarios but this reads like "structs work for any nested object".